tests/_cli/test_sandbox.py

```diff
@@ -37,3 +39,35 @@ def test_no_dependencies():
 app = marimo.App(width="medium")
 """
     assert _get_dependencies(SCRIPT) == []
+
+
+def test_get_local_package_dependencies():
+    SCRIPT = """
+# Copyright 2024 Marimo. All rights reserved.
+# /// script
+# requires-python = ">=3.11"
+# dependencies = [
+#     "pandas @ ~/local/path1",
+#     "numpy @ /local/abs/path2",
+#     "polars",
+#     "-e ~/foo/package1",
+#     "--editable ~/bar/package2",
```

Comment:
i realized trying to install `pandas @ ~/local/path1` and `--editable ~/bar/package2` in the package management UI may break since adding: https://github.com/marimo-team/marimo/pull/2431

Reply:
Fixed in https://github.com/marimo-team/marimo/pull/2448

Reply:
I'll rebase and see if your newest fix solves some of the blockers I was overcoming in my PR, can potentially delete chunks of my code.

Reply:
Depending how you were installing those deps, my change may have fixed some of this